src/main.js

```diff
@@ -86,6 +83,14 @@ async function onGoogleLoad() {
   }).$mount('#app');
 }
 
+async function getSettings(){
+  let response = await axios.get(apiUrl + "/settings")
+  if (response.data){
+    settings = response.data;
+  }
+}
+
+document.onload = getSettings();
```

Comment:
Maybe try running this code inside the async install function.

Reply:
Sounds great, but I'm not sure what that means.

Reply:
OK, I had to double up references to the settings, but that seems to work.